#5152 import public google spreadsheet with editors information

tayowonibi opened the pull request:
resolves https://github.com/elifesciences/issues/issues/5152
- import public editor profile based on new schema
- use same timestamp for all sheets in the spreadsheet imported

tayowonibi committed:
#5152
import public editor profile based on new schema
use same timestamp for all sheets in the spreadsheet imported

tayowonibi committed:
#5152
import public editor profile based on new schema
use same timestamp for all sheets in the spreadsheet imported

de-code commented on flows/editors-google-spreadsheet-deposit/src/editorscsv2json.py:
```diff
@@ -52,4 +54,5 @@ def main(input=None, output=None, filename=None):
 if __name__ == "__main__":
     args = sys.argv[1:]
-    main(filename=first(args) or None)
+    file_name, current_time = get_args(args)
```
I would maybe recommend changing it more to something like _get with default_ (if `argparse` seems too much). (`get_args` doesn't really give away much about what it's doing - we already have `args`)

And then maybe use it a bit like:

```python
file_name = get_at(args, 0, None)
current_time = get_at(args, 1, now())
```

or

```python
current_time = get_at(args, 1, None) or now()
```

tayowonibi replied:
done

de-code commented:
I guess we can't ask NiFi to give us the time in UTC?

tayowonibi committed:
#5152
import public editor profile based on new schema
use same timestamp for all sheets in the spreadsheet imported

tayowonibi committed:
#5152
lint the files
change based on pr

tayowonibi commented:
> I guess we can't ask NiFi to give us the time in UTC?

yes
